src/azure-cli-testsdk/azure/cli/testsdk/base.py

```diff
@@ -297,7 +163,9 @@ def _in_process_execute(self, command):
         try:
             # issue: stderr cannot be redirect in this form, as a result some failure information
             # is lost when command fails.
-            self.exit_code = cli_main(shlex.split(command), file=output_buffer) or 0
+            cmd_parts = [part.replace('"', '')
```

Comment:
Why did you change that? I mean, this is not the point of the port right? PR should be atomic and this one is about "port to azure_devtools". If this is an improvement this is great, but should be in a separate PR. This PR is important enough to avoid this kind of details to be shipped at the same time IMHO.